16_wordle/main_test.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestIsValid(t *testing.T) {
+	word1 := []byte("brown")
+	word2 := []byte("aaaa")
+	word3 := []byte("possa")
+
+	words := [][]byte{word1, word3}
+
+	tests := []struct {
+		word []byte
+		want bool
+	}{
+		{word1, true},
+		{word2, false},
+	}
```

Comment:
I would simplify it like so:

```suggestion
	words := [][]byte{
	  []byte{"brown"},
	  []byte{"aaaa"},
	}

	tests := []struct {
		word []byte
		want bool
	}{
		{[]byte("aaaa"), true},
		{[]byte("possa"), false},
	}
```

Reply:
Similar to the other suggestion. Done